Makefile

```diff
@@ -66,3 +66,7 @@ check_headers:
 .PHONY: generate_docs
 generate_docs:
 	@source test/make.sh && generate_docs
+
+.PHONY: kitchen_all
+kitchen_all:
+	@source kitchen test
```

Comment:
Note for myself - this Make rule looks suspicious, wouldn't this cause the Makefile to attempt to source the output of `kitchen test`, which will choke on invalid bash syntax?

Reply:
Noted --- Will validate and confirm